dotnet/src/Connectors/Connectors.Memory.AzureCosmosDBNoSQL/Connectors.Memory.AzureCosmosDBNoSQL.csproj

```diff
@@ -0,0 +1,30 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <!-- THIS PROPERTY GROUP MUST COME FIRST -->
+    <AssemblyName>Microsoft.SemanticKernel.Connectors.AzureCosmosDBNoSQL</AssemblyName>
+    <RootNamespace>$(AssemblyName)</RootNamespace>
+    <TargetFramework>netstandard2.0</TargetFramework>
```

Comment:
Add `net8.0` ?

Reply:
Is there concrete benefit to doing so when the code doesn't depend on anything .NET 8?  That would require also multi-targeting the `IntegrationTests` project, or accepting test coverage only on .NET 8?

Or do you mean *changing* to `net8.0` instead of `netstandard2.0`?

Reply:
The test projects could multitarget to .NET 8 and .NET 6, but the latter will be out of support in November, and there's rarely an execution difference between the two. The more impactful thing to test would be .NET Framework, for which there's currently no CI support. That's relevant regardless of multitargeting, and is tracked by https://github.com/microsoft/semantic-kernel/issues/6063.

As for concrete benefits, there's the consistency with the rest of the projects, but also that the compiler will see newer surface area, so for example it'll compile string interpolation in a more efficient manner, it'll bind to newer more efficient overloads that might exist, etc.

Reply:
So, just to confirm @stephentoub, your recommendation is to multi-target this project to `netstandard2.0` and `net8.0`, and just leave the test project only testing the `net8.0` implementation?

Reply:
Yes. And then hopefully very soon we get the .NET Framework test leg. @markwallace-microsoft, how hard would that be? @dmytrostruk, you were just mucking around in there... would we be able to add that quickly?

Until then, I get that there's a gap where the netstandard2.0 assets aren't being tested in CI, and that's unfortunate. If we want to try to augment CI with a test leg for .NET 6 before then, and multitarget the test projects to .NET 6, we could do so. I'm just pointing out that this issue isn't unique to this connector; the rest of the SK assemblies are in the same boat.

Reply:
Done.

Reply:
We'll look into extending the testing coverage. Likely this will be after Build.

Reply:
Last night I got all the merged test projects building for net462 and all the unit test projects passing on netfx. https://github.com/microsoft/semantic-kernel/pull/6326 is the draft progress.